tools/deps_info.py

```diff
@@ -196,6 +196,7 @@
 def get_deps_info():
   if not settings.EXCEPTION_HANDLING and settings.LINK_AS_CXX:
     _deps_info['__cxa_begin_catch'] = ['__cxa_is_pointer_type']
+    _deps_info['__cxa_throw'] = ['__cxa_is_pointer_type']
```

Comment:
why is this now needed?

Reply:
This is because `get_exception_ptr` (which calls `__cxa_is_pointer_type`) moved from the old `CatchInfo` to `ExceptionInfo`.   `__cxa_throw` has always depended on `ExceptionInfo` and thus its dependencies increased due to ExceptionInfo inheriting the `__cxa_is_pointer_type` dependencye.

I believe that in practice the overall effect of this change is still to reduce code size for any exceptions-using program, and I have a folloupup commit to further reduce it.

This one dependency regression could be solved by splitting out the `get_exception_ptr` method into is own function?  But for this PR I don't think its worth it.